layers/meta-tegrademo/conf/distro/tegrademo.conf

```diff
@@ -59,3 +59,35 @@ BB_SIGNATURE_HANDLER ?= "OEEquivHash"
 BB_HASHSERVE ??= "auto"
 
 LICENSE_FLAGS_ACCEPTED += "commercial_faad2"
+
+
+#mender
+INHERIT += "mender-full"
```

Comment:
My guess is we'll need to use a dedicated tegrademo-mender.conf if we do it this way again, we won't want this to be a requirement in the base tegrademo.

Ideally it would be great to figure out how to keep this out of the distro conf and build mender support into a dedicated image instead.

Reply:
I agree